runtime/src/genesis_utils.rs

```diff
@@ -196,18 +196,22 @@ pub fn create_genesis_config_with_leader(
 pub fn activate_all_features(genesis_config: &mut GenesisConfig) {
     // Activate all features at genesis in development mode
     for feature_id in FeatureSet::default().inactive {
-        genesis_config.accounts.insert(
-            feature_id,
-            Account::from(feature::create_account(
-                &Feature {
-                    activated_at: Some(0),
-                },
-                std::cmp::max(genesis_config.rent.minimum_balance(Feature::size_of()), 1),
-            )),
-        );
+        activate_feature(genesis_config, feature_id);
     }
 }
 
+pub fn activate_feature(genesis_config: &mut GenesisConfig, feature_id: Pubkey) {
+    genesis_config.accounts.insert(
+        feature_id,
+        Account::from(feature::create_account(
+            &Feature {
+                activated_at: Some(0),
+            },
+            std::cmp::max(genesis_config.rent.minimum_balance(Feature::size_of()), 1),
+        )),
+    );
+}
```

Comment:
There is already a method on `Bank` to activate features; is this new fn needed?

https://github.com/solana-labs/solana/blob/5dbf7d8f91d343ca996b30dc2981691a6c9df443/runtime/src/bank.rs#L7096-L7101

Reply:
This doesn't actually activate the feature, i'm not really sure why this method is present. Basically when you call `compute_active_feature_set` the accounts are scanned and the feature set is updated overwriting any activations done through this method.